buildconfig/Jenkins/Conda/main-pipeline.jenkinsfile

```diff
@@ -1,6 +1,6 @@
 def git_clone(){
     git url: 'https://github.com/mantidproject/mantid.git',
-        branch: 'main_pipeline_prototype'
+        branch: "release-next"
```

Comment:
Is it possible to "inject" this from a job parameter? We're going to want a job for `main` & `release-next` in the future and I think it would be good if we didn't have to change the file.

One other thing would be maybe we consider changing the name of this file? As `main`  is now the default branch name it could be confused that this only relates to that? Perhaps call it `build-and-deploy-pipeline.jenkinsfile` or I'm open to suggestions...

Reply:
`nightly_build_and_deploy.jenkinsfile`

I think pipeline is a bit obsolete, with time we will associate `*.jenkinsfile` with "pipline file"

Reply:
True `pipeline` is really a detail at this point. I like your suggestion.

Reply:
I think
```
def git_clone(){
    git url: 'https://github.com/mantidproject/mantid.git',
        branch: ${env.GIT_BRANCH}
}
```
will give us the same branch that the job specifies in the SCM section. I found it at https://devops.stackexchange.com/a/9574